include/libp2p/multi/content_identifier.hpp

```diff
@@ -34,6 +34,8 @@ namespace libp2p::multi {
     ContentIdentifier(Version version, MulticodecType::Code content_type,
                       Multihash content_address);
 
+    virtual ~ContentIdentifier() = default;
```

Comment:
Why?
I think CID is data class, and not virtual class, it doesnt have virtual methods.
If it's attempt to fix cpp-filecoin clang-tidy warning(or warning-as-error), then cpp-filecoin should be changed (e.g. suppress that warning, or change `CID : ContentIdentifier { ... }` to `CID { ContentIdentifier c; operator ContentIdentifier &(); ... }`).